samples/AutomaticTraceIdInjection/Log4NetExample/Log4NetExample.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Datadog.Trace" Version="1.8.0" />
+    <PackageReference Include="Datadog.Trace" Version="1.23.0" />
     <PackageReference Include="log4net" Version="2.0.8" />
```

Comment:
Can we update `log4net` to version 2.0.10 or higher (current is 2.0.12)? It fixes a [security vulnerability](https://github.com/advisories/GHSA-2cwj-8chv-9pp9).

Reply:
Done in 40d720464c3f5735a49635bb1c5bc2369505a350